keanu-project/src/main/java/io/improbable/keanu/tensor/Tensor.java

```diff
@@ -11,36 +11,42 @@
 
 public interface Tensor<T> {
 
-    int[] SCALAR_SHAPE = new int[]{1, 1};
```

Comment:
The diffs on this file are a bit confusing because stuff has moved about. Can you call out what the significant changes are?

Reply:
No changes. Just moved the static bits to the top, the interface to the middle and the default implementations to the bottom.